README.md

```diff
@@ -73,88 +73,89 @@ The following global settings can be used under the `no-jquery` property to conf
 
 ## 📖 Rules
 
-* [`no-jquery/no-ajax`](docs/no-ajax.md)
-* [`no-jquery/no-ajax-events`](docs/no-ajax-events.md)
-* [`no-jquery/no-and-self`](docs/no-and-self.md)
-* [`no-jquery/no-animate`](docs/no-animate.md)
-* [`no-jquery/no-animate-toggle`](docs/no-animate-toggle.md)
+Where rules are included in the pre-set profiles `slim` or `deprecated-X.X` it is indicated below.
```

Comment:
Should we explain that `1.3` means `deprecated-1.3`?

Reply:
I'm thinking that's what this does.

Reply:
I mean, yes, clearly that's what you're thinking. I'm thinking it doesn't explain it. :-)

Reply:
It lists all the possible pre-set profiles, making it a small logical step to work out what the numbers mean. Feel free to propose something else.